Package.swift

```diff
@@ -10,12 +10,13 @@ let package = Package(
             name: "themis",
             targets: ["themis"]),
     ],
+    // OpenSSL XCF is statically linked to Themis XCF, so no need to have it as a dependency
     dependencies: [],
     targets: [
-        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
-        // Targets can depend on other targets in this package, and on products in packages this package depends on.
         .binaryTarget(name: "themis",
+                      // update version in URL path
                       url: "https://github.com/julepka/themis/releases/download/2.2.1/themis.xcframework.zip",
+                      // Run from package directory:
                       // $swift package compute-checksum output/themis.xcframework.zip
                       checksum: "b4a8d8868cbd09499d75a10315d1231281d40dee11d1fe549cf4893967f3ee38"),
```

Comment:
Thanks for maintenance instructions!

I'd also drop the `$` from `$swift package...` to make this line a bit easier to copy-paste.